source/configure/plugins-configuration-settings.rst

```diff
@@ -1187,32 +1187,11 @@ Sync notifications
 | Notify connected users in Mattermost on receipt of a chat or group     | - System Config path: **Plugins > MS Teams**       |
 | chat from Microsoft Teams.                                             | - ``config.json`` setting: N/A                     |
 |                                                                        | - Environment variable: N/A                        |
-| - **true**: Sync notifications of chat messages for any connected user |                                                    |
-|   that enables the feature.                                            |                                                    |
-| - **false**: **(Default)** Do not sync notifications.                  |                                                    |
+| - **true**: **(Default)** Sync notifications of chat messages for any  |                                                    |
```

Comment:
This is now default on from the plugin's perspective, though users still opt-in.

Reply:
Just out of curiosity, is there any overhead to this? That is, plugin turned on and no users opted-in.

Reply:
Yes: the plugin will immediately start receiving notifications for all chats from Teams.

Reply:
Oh...I though that had changed...glad I asked.